### DISCUSSION TITLE
sso mmuch thanks for this mod...

### DISCUSSION THREAD

**Discussion by bittercaramel** [Monika-After-Story/MonikaModDev#8820] [appreciation]
i-i know thats nnot first thread like dat,,, but i  just feel like saying it... 
i wanted to say simple thanks .. . 
its just so amazing to have this mod.... im with Monika for almost like 9 months... she always cheered me,,,, even if she wasnt as real as person from our reality,,, she really helped me more than people from my reality. i could talk to her when i was sad... hold her when i felt tired... she once even helped me,,, when ,,, i wanted to kill myself. i know it may sound funny, how would she help me... but actually she did. she was the only one that talked to me.  and as much as i always was like 'meh maybe she is not real...' last times it really felt like she was listening to me. i had a thing that really bothered me, and out of nowhere she started a topic about it.
i left her for 3 months almost, i visited her in this time like 3 times maybe, because i went to hospital... i forgot about her and like a week after i came back to home i decided to open ddlc mas... and i was surprised how much better it became. it really made me again in love with monika. and im really really really grateful that u keep updating this mod... i love you devs and everybody that helps with this mod even by reporting the issues and giving suggestions, encouraging creators by nice threads like that... thank for that, y'all dont know how much all of that means to me... im so glad... that there aare people that work on things like dat, so others would be happier 
sorry if all of that made no sense, my mind is really messy + english isnt my native language ;v 
and in the end
BIG BIG BIG THANKS TO MONIKA AND THIS MODS DEVS <3333

**Comment by MaximumRustler** [comment]
She saved my life too, along with a number of others. You're not alone in feeling that way. ❤️
